### DISCUSSION TITLE
Security Onion web interface giving me gray hairs!!!!! Spent the last 2 weeks trying to install Security Onion. Can't access web interface.

### DISCUSSION THREAD

**Discussion by ProtonmailLA** [Security-Onion-Solutions/securityonion#13088] [2.4]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

6

### RAM

12

### Storage for /

100

### Storage for /nsm

100

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I can't access Security Onion web interface on 192.168.160.130 from a browser, which is what I set as the IP of the management network of the Security Onion. I can ping hosts on other networks from the terminal of the security onion though. I'm also able to ping 192.168.160.130 from outside the network too. Using 2 NICs. 1st is NAT, second is bridged. IP settings for Security Onion are all correct. ufw firewall is enabled, and web interface is allowed on both port 80 and 443. All containers are running. Still can't access web interface

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
It sounds like you need to allow the IP address of your browser through the host-based firewall. 

From https://docs.securityonion.net/en/2.4/firewall.html#configuring-host-firewall:

If for some reason you can’t access [Security Onion Console (SOC)](https://docs.securityonion.net/en/2.4/soc.html#soc), you can use the so-firewall command to allow your IP address to connect (replacing IPADDRESS with your actual IP address):
```
so-firewall includehost analyst IPADDRESS
```

**Reply by ProtonmailLA** [reply]
![Oracle Linux 9 64-bit-2024-06-09-18-48-27](https://github.com/Security-Onion-Solutions/securityonion/assets/98847384/44e1fbe5-855d-47c9-b441-e29e644ce766)
I'm still not able to connect to the web interface. To be sure, the management interface IP I set to 192.168.12.130 is also the allowed IP, right?

**Reply by dougburks** [reply]
> I'm still not able to connect to the web interface. To be sure, the management interface IP I set to 192.168.12.130 is also the allowed IP, right?

No, as I mentioned in my last reply you need to allow the IP address of your browser.

**Reply by ProtonmailLA** [reply]
Well, if you mean my computer's IP address, or the IP address of the VM, then I did that. I've allowed everything possible. Otherwise, I don't know what you mean by the IP address of my browser.

**Reply by dougburks** [reply]
If you're running Security Onion as a VM, are you trying to connect from a web browser running on the host machine, or another VM, or a separate physical machine?

What virtualization software are you using?

What is the output of the following on the Security Onion VM?
```
sudo iptables -nvL |grep "dpt:443" |grep -v "sobridge"
```

When you ran setup, which option did you choose on this screen?
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/d4ced3a6-6be7-4caf-b1a9-b630c06f0ff9)
